public/views/address.html

```diff
@@ -16,7 +16,8 @@
       </a>
     </div>
   </div>
-  <h1>Address<small data-ng-if="address.balance >= 0">&nbsp;{{address.balance | currency:currency}} {{currency.symbol}}</small></h1>
+  <h1>Address</h1>
+  <h2>Balance<small data-ng-if="address.balance >= 0">&nbsp;{{address.balance | lisk}} LSK</small></h2>
```

Comment:
`currency:currency` filter is there for a porpose - it allows dynamic display of selected currency, replace it with `lisk` will break that feature. All balances displayed in explorer should using that filter (one and only exception is supply in header - it's always displayed in LSK).